.github/workflows/release.yml

```diff
@@ -0,0 +1,26 @@
+name: Release package
+
+on:
+  release:
+    types: [published]
```

Comment:
To be honest, I understand this. Do I need to?

Reply:
As in; I don't have the knowledge about these options for `on:`.

Reply:
It's not essential to know, no.

This triggers the workflow on publishing a new release in https://github.com/mdn/browser-compat-data/releases. Like triggering on a release tag, except I have a chance to finish writing the release notes beforehand.